LNPopupController/LNPopupController/LNPopupContentView.h

```diff
@@ -34,6 +34,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, strong, readonly) LNPopupCloseButton* popupCloseButton;
 
+/**
+ * Move close button under navigation bars
+ */
+@property (nonatomic) BOOL popupCloseButtonMoveForNavigationBars;
```

Comment:
Do you mind changing the name to `popupCloseButtonAutomaticallyUnobstructsTopBars`?